requirements.txt

```diff
@@ -1,2 +1,8 @@
-Django==1.7.3
+Django >= 1.4, < 1.5
```

Comment:
Travis testing should be fine now, but I'm a bit worried on how realistic these test results are. We are testing component on a different platform than it will be integrated to, and difference in Django from 1.4 to 1.7 seem kind of significant. 
This is probably not the correct place for this discussion, just putting in a note. (I am aware that long term plan is for platform to adopt never version as well)